Vue/README.md

```diff
@@ -0,0 +1,19 @@
+# {{ name }}
+
+> {{ description }}
+
+## Build Setup
+
+```bash
+# install dependencies
+npm install
+
+# serve with hot reload at localhost:8080
```

Comment:
Update readme to be relevant to `parcel`. Running `npm run dev`, this port is not correct.

Reply:
- [x] Fixed in 89c504a